### ISSUE TITLE
Shift + Tab combination doesn't work while holding Q, A or Z

### ISSUE THREAD

**Issue by Gabixel** [ppy/osu#7999]
**Describe the bug:** With any other letter, the <kbd>Shift</kbd> + <kbd>Tab</kbd> in-game combination seems to work as expected. Somehow, it doesn't work while holding <kbd>Q</kbd>, <kbd>A</kbd> or <kbd>Z</kbd>. An easy example (for osu! "standard") could be using <kbd>Z</kbd> on a spinner (with default key combination), and the user wants to show/hide the overlay.

**osu!lazer version:** 2020.225.0

**Comment by peppy** [comment]
I can't reproduce this. Need more details / debug on how this happens.

**Comment by Gabixel** [comment]
Seems that it's not a Lazer issue, but it's caused by my PC.
At first, I thought it was a keyboard issue (it's pretty old: it uses a [PS/2 input](https://en.wikipedia.org/wiki/PS/2_port) 😐).
Then I tried with a wireless one (so it's more recent), same result.
I also used some online key-press testers (and made one, to be 100% sure), and looks like I can't use these buttons while holding <kbd>Shift</kbd>+<kbd>Tab</kbd> (the non-white keys):

![keyboard](https://user-images.githubusercontent.com/43073074/75769010-b809d480-5d45-11ea-9413-7cba08355422.png)
---
tl;dr: It's not Lazer's fault. Somehow my PC doesn't read the input from these keys while holding <kbd>Shift</kbd>+<kbd>Tab</kbd> (I should probably check for updates / see if it's an input issue with the OS).

Since this issue has nothing to do with Lazer, I guess can be closed.

**Comment by bdach** [comment]
As a side note you might be experiencing [a lack of *n*-key rollover](https://en.m.wikipedia.org/wiki/Rollover_(key)). My 6Gv2 for example has 6-key rollover over USB and *n*-key over PS/2. It'll overall most likely depend on hardware (the actual keyboard as well as the port used).
